vnext/Microsoft.ReactNative/Views/ReactRootControl.cpp

```diff
@@ -184,6 +184,7 @@ void ReactRootControl::InitRootView(
   m_SIPEventHandler->AttachView(xamlRootView, /*fireKeyboradEvents:*/ true);
 
   UpdateRootViewInternal();
+  AttachBackHandlers(xamlRootView);
```

Comment:
>AttachBackHandlers [](start = 2, length = 18)

We must also detach in the UninitRootView.

Reply:
Added a function to remove the keyboard accelerators attached to the XAML root element and the BackRequested listener on uninitialization.